### ISSUE TITLE
Nozzle wants to grind through build plate

### ISSUE THREAD

**Issue by Joeyhaas76** [jarrrgh/banana-split#3]
While trying to split a rear tail section of an RC airplane, I used BS hoping it would be a quick, easy solution to the previous issue, which was that the Z axis wanted to dip below the build plate to try printing the section that was dropped below the build plate. I figured BS looked like a simple way to fix this issue, unfortunately the same exact problem occured in that the nozzle wanted to dig into the build plate and grind. How do I avoid this?

**Comment by jarrrgh** [comment]
Hmm, sounds odd. How does the preview look after slicing? Does it look like it would print below the build plate?

**Comment by Joeyhaas76** [comment]
Preview looks normal, except that it still shows the half below the print
bed, but the preview shows the actual printing start at the build plate.
Unfortunately, when it actually starts printing, the nozzle is grinding
into the bed. Fortunately for the first have, I adjusted the Z axis on the
fly, raised it about .5-.6mm and it continued to print fine. Can you verify
that I’m using BS correctly? So I just select the model, then click the BS
icon, then select the cut button and it splits the model in half and places
the other half on the bed right next to the first half. This is where I
noticed that the bottom halves of each section were still showing under the
bed, and couldn’t figure out how to get rid of them, so I sliced it and
even after slicing I still saw the bottom halves were still showing under
the bed. I tried to print at that point and obviously something wasn’t
right when the nozzle started digging into the bed. Is there a way to
remove the bottom half from underneath the bed?

**Comment by jarrrgh** [comment]
Yes, that sounds like correct behavior. Rather than actually splitting the model, it's just duplicating the object and flipping it around. It's what people used to do manually, but now it's just automated and easier to adjust later.

Maybe there's something wrong with bed leveling or maybe the bed is slightly warped? I had a warped bed with my Ender 3 Pro. Edges were way higher than the center, and if I leveled it according to the center, the nozzle was grinding the plate on the edges. If I leveled it according to the edges, the nozzle was just printing in the air. I fixed it with some copper tape 😄 Of course better solution would have been automatic bed leveling.

**Comment by Joeyhaas76** [comment]
I do have auto bed leveling, and the bed is good. It doesn’t do that with
any other prints, only when I try to split them, but like I said, If I
adjust the height on the fly, it solved the problem, just not sure why it
wants to grind only when I split the model.
